ubuntu.sh

```diff
@@ -236,8 +235,9 @@ ubuntu_grub_fix() {
     local volgroup=$(echo $line | cut -d " " -f9)
     local dmdevice=$(echo $line | cut -d "/" -f2)
 
-    rm $mapper/$volgroup
-    cp -R $FOLD/hdd/dev/$dmdevice $mapper/$volgroup
+    rm "$mapper/$volgroup"
```

Comment:
probably not needed anymore, we've got https://github.com/virtapi/installimage/issues/46 for this.